### PR TITLE
add templates and sources and fix

### LINKED ISSUE #130: tambahkan neumanga 🎉
### Source name

Neumanga

### Source link

https://neumanga.net/

### Language

indonesia

### Other details

device oppo android 5.1 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have checked that the source does not already exist on the app.
- [X] I have checked that the source does not already exist by searching the [GitHub repository](https://github.com/KotatsuApp/kotatsu-parsers) and verified it does not appear in the code base.

### PR DESCRIPTION
**Pull request by davvarrr**
add templates and small fix on mmrcms and rename bad typo val

manhwa18.com
klz9.com
welovemanga.one
lkscanlation.com
weloma.art
saytruyenhay.com
www.witcomics.net
otsugami.id
alceascan.my.id
beehentai.com
mangabuddy.com
mangacute.com
mangaforest.me
mangapuma.com
mangaxyz.com
toonitube.com
toonily.me
truemanga.com
xoxocomics.net
www.nettruyenmax.com
nhattruyenmin.com
scansmangas.me
scanmanga-vf.ws
hensekai.com
www.maid.my.id
komikindo.info
shirodoujin.com
neumanga.net -> close #130 
papscan.com
komikzoid.xyz
neumanga.xyz
sektekomik.xyz
h.mangabat.com
mangairo.com
manganato.com -> part of #171

### CODE REVIEW

**Comment by davvarrr** [comment]
For MangaboxParser I could not find a way to take the 2nd domain so I created a val but if there was a posibiliter change it :)

as usual, there are probably some mistakes, sorry in advance.

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/fr/ScansMangasMe.kt [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+import org.koitharu.kotatsu.parsers.PagedMangaParser
+import org.koitharu.kotatsu.parsers.config.ConfigKey
+import org.koitharu.kotatsu.parsers.model.*
+import org.koitharu.kotatsu.parsers.network.UserAgents
+import org.koitharu.kotatsu.parsers.util.*
+import java.util.*
+
+@MangaSourceParser("SCANS_MANGAS_ME", "Scans Mangas Me", "fr")
+internal class ScansMangasMe(context: MangaLoaderContext) :
+	PagedMangaParser(context, MangaSource.SCANS_MANGAS_ME, 1000000) {
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/fr/ScansMangasMe.kt:19]: `1000000`?
Does it means this source has only one page?
I guess in these cases might be better to use `MangaParser` and return an empty list if `offset !=0`

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/fr/ScansMangasMe.kt [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+			author = aut,
+			state = state,
+			chapters = chaptersDeferred,
+			isNsfw = manga.isNsfw,
+		)
+	}
+
+
+	private fun getChapters(doc: Document): List<MangaChapter> {
+		return doc.body().select("ul#chapter_list li").mapChapters(reversed = true) { i, li ->
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/fr/ScansMangasMe.kt:146]:
```suggestion
		return doc.body().requireElementById("chapter_list").select("li").mapChapters(reversed = true) { i, li ->
```

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/zmanga/ZMangaParser.kt [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+					title = a.text().toTitleCase().replace(",", ""),
+					source = source,
+				)
+			},
+			description = desc,
+			altTitle = alt,
+			author = aut,
+			state = state,
+			chapters = chaptersDeferred.await(),
+			isNsfw = nsfw || manga.isNsfw,
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/zmanga/ZMangaParser.kt:177]:
```suggestion
			isNsfw = manga.isNsfw || doc.getElementById("adt-warning") != null,
```

Small optimization)

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/fr/ScansMangasMe.kt (hunk 1 above)
Comment (davvarrr) [reply src/main/kotlin/org/koitharu/kotatsu/parsers/site/fr/ScansMangasMe.kt:19]: Yes, it's only one page, but I put that in so as not to be emersed during the parse.
I let you correct it I could not do it before several days.
Tanks

Comment (davvarrr) [reply src/main/kotlin/org/koitharu/kotatsu/parsers/site/fr/ScansMangasMe.kt]: Finally I was able to do it quickly evnet to take care of something else, I'm not sure it's what you ask but do not hesitate to change if necessary.

**Review by Koitharu — APPROVED** [review]

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/animebootstrap/AnimeBootstrapParser.kt [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+				)
+			}
+
+			return pages
+		}
+
+	}
+
+
+	protected fun Element.src(): String? {
```
[the remark this answers is not in the document]
Comment (davvarrr) [reply src/main/kotlin/org/koitharu/kotatsu/parsers/site/animebootstrap/AnimeBootstrapParser.kt]: It's already done look at 41fa80e

**Pull request merged**